rosdep/python.yaml

```diff
@@ -3647,6 +3647,11 @@ python-sqlalchemy:
     wily: [python-sqlalchemy]
     xenial: [python-sqlalchemy]
     xenial_python3: [python3-sqlalchemy]
+python-subprocess32:
+  debian: [python-subprocess32]
```

Comment:
This package is not available on jessie, so we should explicitely set the rule to `null` for that platform